airbyte-server/Dockerfile

```diff
@@ -13,6 +13,9 @@ RUN chmod +x wait
 
 COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 
+RUN mkdir latest_seeds
+COPY build/resources/main/config latest_seeds
```

Comment:
why does this copy into `latest_seeds` and `seed.Dockerfile` copies into `latest_seeds/config?

Reply:
seed.Dockerfile is populating the `data` volume and we want the data to be present in `/data/config` path in the volume. This is just the latest_seeds and is only required for automatic migration